test/tables.js

```diff
@@ -82,4 +82,12 @@ describe('Tables controller', () => {
 
   });
 
+  it('should recuce all results to one number', function *test6() {
+    yield tables.getTables();
+    winston.debug(tables[0].game.players.situation);
+    tables[0].game.players.situation.should.be.an('number');
```

Comment:
`players` is an array, I have doubts it should work like this. Will check it to be sure 👶